.github/workflows/ci.yml

```diff
@@ -40,12 +40,19 @@ jobs:
             redis-version: 'latest'
             python-version: '3.10'
 
-          # TODO: re-enable with https://github.com/jazzband/django-redis/issues/569
           # pre-release Django and redis
           - django-version: 'main'
             redis-version: 'master'
             python-version: '3.10'
 
+        # exclude python 3.6 and 3.7 for django 4.x
```

Comment:
Maybe a rationale on why those are excluded would make this perfect…

Reply:
They are not supported, maybe they are using new syntax or are looking forward to it. I will write something

Reply:
Thanks!